demo/src/stories/input.stories.jsx

```diff
@@ -17,21 +17,21 @@ export const Input1 = () => (
 );
 Input1.storyName = 'Input with label (should pass)';
 
-export const Input2 = () => <Input placeholder="Favorite coffee" />;
+export const Input2 = () => <Input />;
```

Comment:
Placeholders now count as labels for an input, so I had to remove them to get this to actually fail.